src/core/remote.js

```diff
@@ -1610,7 +1610,7 @@ Remote.prototype.requestBookOffers = function(options_, callback_) {
   const request = new Request(this, 'book_offers');
 
   request.message.taker_gets = {
-    currency: Currency.json_rewrite(gets.currency, {force_hex: true})
```

Comment:
I guess a better experiment would be
'Currency.json_rewrite(interestBearing, {force_hex: true})'. With force_hex
both true and false.

You can check but 99% sure rippled only understands hex (likely reason the
force b4) Arguably to_json() should speak rippled. That rate suffix is for
humanoids.